File: 9주/타겟 넘버/ㄴㄹ/target_num.py
def solution(numbers, target):
    answer = []
    
    def recur(num,tar,i,hap):
        if i>=len(num):
            if hap==tar:
                answer.append(hap)
            return
        recur(num,tar,i+1,hap+num[i])
        recur(num,tar,i+1,hap-num[i])
    
    recur(numbers,target,0,0)
    return len(answer)

File: 9주/타겟 넘버/ㅇㅎ/target_num.js
function solution(numbers, target) {
    let cnt = 0;
    
    getNum(0,0);
    function getNum(idx, sum){
        if(idx === numbers.length){
            if(sum === target){
                cnt++;
            }
            return;
        }
        getNum(idx+1, sum+numbers[idx]);
        getNum(idx+1, sum-numbers[idx]);
    }
    return cnt;
}
